src/fs.rs

```diff
@@ -18,79 +17,70 @@ use crate::{RulesetAttr, RulesetCreatedAttr};
 #[cfg(test)]
 use strum::IntoEnumIterator;
 
-/// File system access right.
-///
-/// Each variant of `AccessFs` is an [access right](https://www.kernel.org/doc/html/latest/userspace-api/landlock.html#access-rights)
-/// for the file system.
-/// A set of access rights can be created with [`BitFlags<AccessFs>`](BitFlags).
-///
-/// # Example
-///
-/// ```
-/// use landlock::{ABI, Access, AccessFs, BitFlags, make_bitflags};
-///
-/// let exec = AccessFs::Execute;
-///
-/// let exec_set: BitFlags<AccessFs> = exec.into();
-///
-/// let file_content = make_bitflags!(AccessFs::{Execute | WriteFile | ReadFile});
-///
-/// let fs_v1 = AccessFs::from_all(ABI::V1);
-///
-/// let without_exec = fs_v1 & !AccessFs::Execute;
-///
-/// assert_eq!(fs_v1 | AccessFs::Refer, AccessFs::from_all(ABI::V2));
-/// ```
-///
-/// # Warning
-///
-/// To avoid unknown restrictions **don't use `BitFlags::<AccessFs>::all()` nor `BitFlags::ALL`**,
-/// but use a version you tested and vetted instead,
-/// for instance [`AccessFs::from_all(ABI::V1)`](Access::from_all).
-/// Direct use of **the [`BitFlags`] API is deprecated**.
-/// See [`ABI`] for the rationale and help to test it.
```

Comment:
I've removed this warning as `Access*::all()` is no longer part of the public api.